en/gps_compass/rtk_gps_gem1305.md

```diff
@@ -0,0 +1,144 @@
+# DATAGNSS GEM1305 RTK Receiver with antenna
+
+GEM1305 is a RTK receiver with antenna designed and manufactured by [DATAGNSS](https://www.datagnss.com/).
+
+GEM1305 is based on the new generation CYNOSURE IV dual-core GNSS SoC.
+It supports RTK functionality with a maximum data update rate of 10Hz.
+
+<img src="../../assets/hardware/gps/datagnss_gem1305/datagnss-gem1305-02.png" width="500px" alt="DATAGNSS GEM1305 RTK Receiver">
+
+::: info
+
+- GEM1305 RTK doesn't support moving base mode yet.
+- GEM1305 RTK does not include a compass.
+
+:::
+
+## Where to Buy
+
+- [GEM1305 RTK receiver with antenna](https://www.datagnss.com/collections/gnss-for-drone/products/gem1305) (www.datagnss.com)
+
+  <img src="../../assets/hardware/gps/datagnss_gem1305/gem1305_hero.png" width="500px" alt="DATAGNSS GEM1305 RTK Receiver">
+
+- [DGM10 RTK Receiver](https://www.datagnss.com/collections/gnss-for-drone/products/dgm10-rtk-receiver) (same receiver in a casing).
+
+  ![dgm10_rtk_receiver.png](../../assets/hardware/gps/datagnss_gem1305/dgm10_rtk_receiver.png)
```

Comment:
Inspecting your site, this appears to be the same product in a case - is that correct?

Reply:
yes, it is correct.